browser/react/components/Login/Signup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default (props) => {
+  console.log('PROPS', props);
+  return (
+    <div style={props.styles.signUpContainer}>
+      <form onSubmit={props.signup}>
+        <div className="form-group">
+          <input
+            key="name"
+            name="name"
+            type="name"
```

Comment:
I don't think this is a type with any semantic meaning. E.g. email and password does. You can see the types at [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input )